lib/utils/groupMessages.js

```diff
@@ -20,6 +20,7 @@ var GroupMessages = {
     RegisterTemplateDeprecationWarnings: 'Register Template Deprecation Warnings',
     SassMapFormatDeprecationWarnings: 'Sass Map Format Deprecation Warnings',
     MissingRegisterTransformErrors: 'Missing Register Transform Errors',
+    PropertyNameCollisionWarnings: 'Property Name Collision Warnings',
```

Comment:
We are getting some real use out of the groupMessages! I never thought we'd have this many warnings and errors, but it follows our tenet to fail loudly! Super cool.